lib/driver.rb

```diff
@@ -43,5 +44,67 @@ def add_trip(trip)
 
       @trips << trip
     end
+
+    def get_total_revenue
+      total_rev = 0.0
+      @trips.each do |trip|
+        if trip.cost != nil
```

Comment:
In Ruby, `nil` is a "falsey" value, as is `false`. Everything else evaluates as truthy.

https://gist.github.com/jfarmer/2647362

You can just write:

```ruby
if trip.cost
  # ...
end
```

Reply:
√